src/utils/openapi.ts

```diff
@@ -368,6 +368,17 @@ export function isNamedDefinition(pointer?: string): boolean {
   return /^#\/components\/schemas\/[^\/]+$/.test(pointer || '');
 }
 
+function humanizeMultipleOfConstraint(multipleOf: number | undefined): string | undefined {
+  if (multipleOf === undefined) {
+    return;
+  }
+  const strigifiedMultipleOf = multipleOf.toString(10);
+  if (!/^0\.0*1$/.test(strigifiedMultipleOf)) {
+    return `multiple of ${multipleOf}`;
+  }
+  return `decimals <= ${strigifiedMultipleOf.split('.')[1].length} digits`;
```

Comment:
@nanov @adamaltman What do you think if we change it to `n decimals max` instead of `decimals <= n`?

Reply:
Well, I have nothing against it.

The only thing i might thing of is that we may provide those hardcoded strings as configurations options to allow internationalisation.

Reply:
having this in mind having strings as ```decimals <= {{x}}``` are some how easier to manage, translation wise speaking without introduction of (simple) templating system or a i18n solution.

Reply:
These are both wrong actually.  I'm glad that you brought it up.

Decimals is the concept.  Decimal, from the dictionary:

> a fraction whose denominator is a power of ten and whose numerator is expressed by figures placed to the right of a decimal point.

What we're talking about here specifically is called decimal places. 

So it should be like this:

* decimal places <= n
* n decimal places max

However, in the second phrase, `n decimal places max` the places must turn singular if `n == 1`.

Reply:
If we decide to go with the second option we could go with the good old:

* n decimal place(s) max

As some languages have also a double form ( ie. singular, double, and plural ) i think it is better to use simpler descriptions in order to make internationalization simpler, with minimal overhead when the time comes.

Reply:
I've taken your first suggestion and implemented it, I took the time to see how much effort would it be to implement mini translating system, and it seems as a straight forward task ( can work similar to the theme option ) and for our use case it can be done with zero-dependencies.

The question here is should i implement it here, or do a new PR for it?